docs/getting_started/modin_vs_dask_vs_koalas.rst

```diff
@@ -0,0 +1,91 @@
+Modin vs. Dask DataFrame and Modin vs. Koalas
+===============================================
+
+Libraries such as Dask DataFrame (DaskDF for short) and Koalas aim to support the pandas API on top of distributed computing frameworks, Dask and Spark respectively. Instead, Modin aims to preserve the pandas API and semantics as is, while abstracting away the details of the distributed computing framework underneath. Thus, the aims of these libraries are fundamentally different.
+
+Specifically, Modin enables pandas-like
+
+* row and column-parallel operations, unlike DaskDF and Koalas that only support row-parallel operations
+* indexing & ordering semantics, unlike DaskDF and Koalas that deviate from these semantics
+* eager execution, unlike DaskDF and Koalas that provide lazy execution
```

Comment:
```suggestion
* row and column-parallel operations, unlike DaskDF and Koalas which only support row-parallel operations
* indexing & ordering semantics, unlike DaskDF and Koalas which deviate from these semantics
* eager execution, unlike DaskDF and Koalas which provide lazy execution
```

Reply:
Unclear what `indexing & ordering semantics` means without prior context, can we say `indexing & ordering capabilities` instead?

Reply:
If we are changing that -> which, we should also add a comma before each which.

Re: semantics vs. capabilities, it is not really about capabilities but it is about what is the expected behavior (hence semantics). We can change it to "behavior" if semantics is hard to understand.

Reply:
Added comma before which.

Reply:
I agree "behavior" is a better way of framing this, semantics can mean many things.